### PR TITLE
Feature/compact table display

### PR DESCRIPTION
**Pull request by evencewang**
Made final touches to the compact table. Took out the const concise, and changed to this.opts.concise

### CODE REVIEW

**Review by jimmy-lan — CHANGES_REQUESTED** [review]
Hey Evence,

Thanks for contributing! Just want to bump on the other comment from pull request https://github.com/lanyanxiang/habits-cli/pull/76, if you have a moment to take a look.

**Comment by evencewang** [comment]
> Hey Evence,
> 
> Thanks for contributing! Just want to bump on the other comment from pull request #76, if you have a moment to take a look.

Hey Jimmy, I am assuming you are talking about the listed 0 transactions thing, if that is the case I have changed it back.

**Comment by jimmy-lan** [comment]
Hey Evence, yes, I was talking about this review: https://github.com/lanyanxiang/habits-cli/pull/76#pullrequestreview-995975709. Let's revert the copy change for now (it was written this way based on a user's recommendation, but if we get data that suggests otherwise, we can change the copy by then).

**Comment by evencewang** [comment]
Alright sentence just changed back.

File: src/commands/transaction/list.ts [hunk 1]
```diff
@@ -27,10 +27,34 @@ interface PromptAnswers {
...
+    }
+  );
+  display.table.print(table);
+};
+
 const displayTransactions = (response: SuccessResponse) => {
   const transactions = response.data.payload as ListResponsePayload;
   if (!transactions.length) {
-    return console.log("Listed 0 transactions.");
+    return console.log(chalk.cyan("Listed 0 transactions."));
```
Comment (jimmy-lan) [inline src/commands/transaction/list.ts]: Just this one `chalk.cyan` left - would you mind quickly fix this?

Comment (evencewang) [reply src/commands/transaction/list.ts]: Removed chalk.cyan. Think it should be good now. Please let me know if there is anything else.

**Comment by evencewang** [comment]
Conflicts just resolved!

**Review by jimmy-lan — APPROVED** [review]
Thanks Evence!

**Pull request merged**